src/js/authPage.js

```diff
@@ -14,18 +15,24 @@ export default class AuthPage extends Component {
     switch (type) {
     case 'login':
       request.post('/auth/login')
-             .send({email, password})
+             .send({name, email, password})
              .end((err, res) => {
                if (err) this.setState({error: true});
-               else this.props.login(res.body.token)
+               else {
+                 username = res.body.name;
+                 this.props.login(res.body.token, username, email)
```

Comment:
Make this and the above one name and dont use undeclared vars. Use `var { name } = res.body`